catalogue/webapp/components/ImageCard/ImageCard.tsx

```diff
@@ -50,6 +50,7 @@ const ImageCard: FunctionComponent<Props> = ({
           if (onClick) onClick(event);
         }}
         id={id}
+        data-gtm-trigger={isEnhanced ? 'open_image_modal' : undefined}
```

Comment:
do we need this conditional here?

Reply:
it wouldn't open the image modal if `isEnhanced` was `false` because the link would just behave as a link, so I think the conditional makes sense